scss/detail.scss

```diff
@@ -64,6 +64,13 @@ $lower-screen-width: calc(#{$book-badge-width} * 0.7);
   height: 300px;
   margin: 15px;
   width: 200px;
+  transition: 0.2s ease-out;
+
+
```

Comment:
Remove one empty line

Reply:
I dont see this empty line. In the 69 is the `&hover` :thinking: 

I make the commit, lets check this out now!

Reply:
Perfect